.gitignore

```
@@ -108,4 +108,16 @@ m4/lt~obsolete.m4
 Makefile
 
 # IDEs
-.idea
\ No newline at end of file
+.idea
+
+# tests
+tests/regression/server_root/conf/*t*.conf
+tests/regression/server_root/upload/*-file-*
+tests/regression/server_root/data/*-ip.*
+tests/regression/server_root/logs/*.log
+tests/regression/server_root/logs/audit/*/
```

Ignore all paths below `server_root`. Will not affect already tracked files. Also, you can still add a path that is ignored using `git add` if you need to (ignored paths will not be part of `git add .`).
```suggestion
tests/regression/server_root/**
```

---

Okay, let's see. (Although I like much better to add the minimal patterns, and we **definitely** must care of this directory in the future.)